Formula/trino.rb

```diff
@@ -0,0 +1,60 @@
+class Trino < Formula
+  desc "Distributed SQL query engine for big data"
+  homepage "https://trino.io"
+  url "https://search.maven.org/remotecontent?filepath=io/trino/trino-server/360/trino-server-360.tar.gz"
+  sha256 "cd313867fd5461a26920ca6cb170413893853b0cc1acc4f454d4141a0790a50e"
+  license "Apache-2.0"
+
+  livecheck do
+    url "https://search.maven.org/remotecontent?filepath=io/trino/trino-server/"
+    regex(%r{href=["']?v?(\d+(?:\.\d+)*)/?["' >]}i)
+  end
+
+  depends_on "openjdk"
+
+  resource "trino-src" do
+    url "https://github.com/trinodb/trino/archive/360.zip"
```

Comment:
This fails the audit: `Use GitHub tarballs rather than zipballs`

However, the tarball seems to have a corrupted file:
```
Error: Failure while executing; `tar --extract --no-same-owner --file /Users/Branch/Library/Caches/Homebrew/downloads/d3a1522f0b88c09af3b4768783f9a0f6012f26d36e66189f6f198f15f002b426--trino-360.tar.gz --directory /private/tmp/d20210813-21348-1lnvrom` exited with 1. Here's the output:
trino-360/plugin/trino-hive/src/test/resources/spark_bucketed_nation/.part-00000-0a72da44-98a5-4b85-a513-2b2e82ec274f_00000.c000.snappy.orc.crc: Failed to restore metadata
tar: Error exit delayed from previous errors.
```

any potential workarounds here?

Reply:
@Homebrew/brew, thoughts?

Reply:
Wait, the github generated tarbal is corrupted?

Reply:
Indeed, see more info at https://github.com/trinodb/trino/issues/8877. Note that this seems to only be an issue with bsd tar, not gnu tar. Assuming there's no way to tell the formula to use the latter?

Reply:
We could hack it with `using: :no_unzip` and then untar it ourselves with `gnu-tar` in the `install` method. This is a terrible hack, though.

Reply:
that indeed works. which do you prefer, using `gnu-tar` or downgrading the version?

Reply:
Let's use `gnu-tar`, but document it with comments explaining why and include a link to the upstream issue, please.